slometricsemitter/config.go

```diff
@@ -0,0 +1,57 @@
+package slometricsemitter
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"slices"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	SloConfigFile string `mapstructure:"slo_config_file"`
```

Comment:
Nit picking, but I would rather see SLO capitalized everywhere because I keep on pronoucing it as "slow" in my head.
```suggestion
	SLOConfigFile string `mapstructure:"slo_config_file"`
```

Reply:
i get that lol i'll rename